app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    people_mapping = {}
```

Comment:
There is no need to have the `people_mapping` variable here, it duplicates the functionality of the `Person.people` attribute. Remove the `people_mapping` variable and the logic that is related to it.